Tests/survey/Test-PackageTrustVerificationContracts.ps1

```
@@ -0,0 +1,349 @@
+            $true
+        )
+    )
+    $ekuOids = [System.Security.Cryptography.OidCollection]::new()
+    [void]$ekuOids.Add([System.Security.Cryptography.Oid]::new('1.3.6.1.5.5.7.3.3', 'Code Signing'))
+    $request.CertificateExtensions.Add(
+        [System.Security.Cryptography.X509Certificates.X509EnhancedKeyUsageExtension]::new($ekuOids, $false)
+    )
+    $notBefore = [DateTimeOffset]::UtcNow.AddMinutes(-5)
+    $notAfter = $notBefore.AddDays(2)
+    $ephemeral = $request.CreateSelfSigned($notBefore, $notAfter)
+    $securePassword = ConvertTo-SecureString -String ('fixture-' + [guid]::NewGuid().ToString('N')) -AsPlainText -Force
```

PSScriptAnalyzer (1.25.0)

[error] File 'Test-PackageTrustVerificationContracts.ps1' uses ConvertTo-SecureString with plaintext. This will expose secure information. Encrypted standard strings should be used instead.

(PSAvoidUsingConvertToSecureStringWithPlainText)